src/examples/UsePreferredLanguageExample.tsx

```diff
@@ -0,0 +1,13 @@
+import usePreferredLanguage from "../hooks/usePreferredLanguage";
+
+export default function App() {
```

Comment:
polish: could you rename this `UsePreferredLanguageExample` please ?

Reply:
Oops, got it